src/create.cpp

```diff
@@ -863,6 +868,40 @@ namespace create {
     }
   }
 
+  float Create::getLeftWheelDistance() const {
+      return totalLeftDist;
+  }
+
+  float Create::getRightWheelDistance() const {
+      return totalRightDist;
+  }
+
+  float Create::getLeftWheelVel() const {
```

Comment:
Note that according the [Create OI Spec](http://www.irobot.com/filelibrary/pdfs/hrd/create/Create%20Open%20Interface_v2.pdf) this will report the _requested_ velocity, not a measured value. I would rename this function to `getRequestedLeftWheelVel()` if you'd like to have that information (same for right wheel). If you want the measured velocity you can compute it above in the `onData()` function after computing the wheel distances and using `dt`.